src/tools/compiletest/src/json.rs

```diff
@@ -1,5 +1,4 @@
 //! These structs are a subset of the ones found in `rustc_errors::json`.
-//! They are only used for deserialization of JSON output provided by libtest.
```

Comment:
Remark: you are right, this is wrong; it's not libtest, it's not just compiler, it's also rustdoc and maybe even more.